pkg/scheduler/conf/scheduler_conf.go

```diff
@@ -19,16 +19,25 @@ package conf
 // SchedulerConfiguration defines the configuration of scheduler.
 type SchedulerConfiguration struct {
 	// Actions defines the actions list of scheduler in order
-	Actions string `yaml:"actions"`
+	Actions []SchedulerAction `yaml:"actions"`
 	// Tiers defines plugins in different tiers
 	Tiers []Tier `yaml:"tiers"`
 }
 
+// BackfillFlagName the flag to enable backfill for non-best effort jobs
+const BackfillFlagName = "enabledNonBestEffort"
```

Comment:
How about name it as `backfillNonBestEffortJobs`?

Reply:
do you mean the variable name or the string?

Reply:
I was referring to rename `enableNonBestEffort` to `backfillNonBestEffortJobs`.

Reply:
fixed